init.sh

```diff
@@ -80,6 +80,7 @@ cd srcomp-cli
 cd ..
 cd srcomp-screens
     bower install
+    sed 's_localhost/_localhost:5112/_' config.example.json >config.json
```

Comment:
This seems fairly brittle -- if the example config changes (perhaps to include a port itself) then will silently stop working. Is there a better way we can express this?

Reply:
To be able to handle a port being specified we could do:
```suggestion
    sed 's_localhost.*/_localhost:5112/_' config.example.json >config.json
```
Alternatively we could fill the full path, there's some slight weirdness to handle trailing commas:
```suggestion
    sed -E 's_"apiurl":\s*".*(".*)_"apiurl": "http://localhost:5112/comp-api\1_;
        s_"streamurl":\s*".*(".*)_"streamurl": "http://localhost:5112/stream\1_' config.example.json >config.json
```

Reply:
I think I prefer the first of these, though perhaps the match should be `[^/]*`?

Reply:
Just realised that this regex gives the wrong result, since stream is actually on a different port we want:
```json
{
  "apiurl": "http://localhost:5112/comp-api",
  "streamurl": "http://localhost:5001/"
}
```
not: 
```json
{
  "apiurl": "http://localhost:5112/comp-api",
  "streamurl": "http://localhost:5112/stream"
}
```
For this situation I feel regex may not be the best tool.

Reply:
I've switched to inline python in acf258edc09be4c68f1a49c5f627424731c93b3c